src/webots/nodes/WbTriangleMeshGeometry.hpp

```diff
@@ -0,0 +1,141 @@
+// Copyright 1996-2020 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WB_TRIANGLE_MESH_GEOMETRY_HPP
+#define WB_TRIANGLE_MESH_GEOMETRY_HPP
+
+#include "WbGeometry.hpp"
+#include "WbTriangleMeshCache.hpp"
+
+#include <unordered_map>
+
+class WbCoordinate;
+class WbNormal;
+class WbTextureCoordinate;
```

Comment:
Are this class include really needed?
It doesn't seem these classes are used here